### DISCUSSION TITLE
Project Generation

### DISCUSSION THREAD

**Discussion by samchouse** [prisma/prisma#6192] [Q&A] [answered]
Is there a command for Prisma that does the same as `typeorm init --name server --database postgres`?

**Comment by ryands17** [comment, marked as answer]
@Xenfo 👋 
We have `prisma init` as a [bootstrapping command](https://www.prisma.io/docs/reference/api-reference/command-reference#init).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
